tsconfig.build.json

```diff
@@ -0,0 +1,36 @@
+{
+  "compilerOptions": {
+    "target": "es5",
+    "module": "commonjs",
+    "allowJs": true,
+    "checkJs": true,
+    "jsx": "react-jsx",
+    "declaration": true,
+    "declarationMap": true,
+    "sourceMap": true,
+    "removeComments": true,
+    "downlevelIteration": true,
+
+    "strict": true,
+    "noImplicitAny": true,
+    "strictNullChecks": true,
+    "strictFunctionTypes": true,
+    "strictBindCallApply": true,
+    "strictPropertyInitialization": true,
+    "noImplicitThis": true,
+    "alwaysStrict": true,
+
+    "noUnusedLocals": true,
+    "noUnusedParameters": false,
```

Comment:
`noUnusedParameters` should be enabled since `ts(6133)` and `@typescript-eslint/no-unused-vars` enforce no unused parameters (and there isn't really a reason to allow it anyway)

Reply:
sometimes you might have a callback where you for example need the 3rd parameter, and IMO it's nicer to just name them out instead of having `_, _, something` for example, which is why I figured this could be disabled.